ljepotaservis/ljepotaservis.Web/ClientApp/src/components/superAdminSide/AddStore/AddOwner.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+
+class AddStore extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Maybe this.state = new userDto could work perfectly here. Just to reduce retyping and to stay dry.